builtin/mainmenu/content/contentdb_api.lua

```diff
@@ -0,0 +1,58 @@
+--Minetest
+--Copyright (C) 2023 rubenwardy
+--
+--This program is free software; you can redistribute it and/or modify
+--it under the terms of the GNU Lesser General Public License as published by
+--the Free Software Foundation; either version 2.1 of the License, or
+--(at your option) any later version.
+--
+--This program is distributed in the hope that it will be useful,
+--but WITHOUT ANY WARRANTY; without even the implied warranty of
+--MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+--GNU Lesser General Public License for more details.
+--
+--You should have received a copy of the GNU Lesser General Public License along
+--with this program; if not, write to the Free Software Foundation, Inc.,
+--51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+if not core.get_http_api then
+	return
+end
+
+contentdb_api = {}
```

Comment:
I assume this function will get other methods in the future?
The "contentdb_api" (why not "cdb_api" anyway?) name is very general.

Reply:
Yeah, I'm hesitant to move other things to here in this PR as it would conflict with Roller's work on the CDB dialog redesign. Perhaps I should move this code for `update_detection.lua` in the meantime

Reply:
> then a refactor could be done more holistically in the future

Sure